node/src/accountant/payment_adjuster/disqualification_arbiter.rs

```diff
@@ -157,78 +150,162 @@ impl DisqualificationGauge for DisqualificationGaugeReal {
         threshold_intercept_minor: u128,
         permanent_debt_allowed_minor: u128,
     ) -> u128 {
+        // This signs that the debt lies in the horizontal area of the payment thresholds, and thus
+        // should be paid in the whole size.
         if threshold_intercept_minor == permanent_debt_allowed_minor {
             return account_balance_minor;
         }
-        let exceeding_debt_part = account_balance_minor - threshold_intercept_minor;
-        if DisqualificationGaugeReal::qualifies_for_double_margin(
+        Self::determine_adequate_minimal_payment(
             account_balance_minor,
             threshold_intercept_minor,
             permanent_debt_allowed_minor,
-        ) {
-            exceeding_debt_part + 2 * permanent_debt_allowed_minor
-        } else {
-            exceeding_debt_part + permanent_debt_allowed_minor
-        }
+        )
     }
 }
 
 impl DisqualificationGaugeReal {
-    const FIRST_CONDITION_COEFFICIENT: u128 = 2;
-    const SECOND_CONDITION_COEFFICIENT: u128 = 2;
-    fn qualifies_for_double_margin(
+    const FIRST_QUALIFICATION_CONDITION_COEFFICIENT: u128 = 2;
+    const SECOND_QUALIFICATION_CONDITION_COEFFICIENT: u128 = 2;
+    const MULTIPLIER_FOR_THICKER_MARGIN: u128 = 2;
+
+    fn qualifies_for_thicker_margin(
         account_balance_minor: u128,
         threshold_intercept_minor: u128,
         permanent_debt_allowed_minor: u128,
     ) -> bool {
         let exceeding_threshold = account_balance_minor - threshold_intercept_minor;
         let considered_forgiven = threshold_intercept_minor - permanent_debt_allowed_minor;
-        let minimal_payment_accepted = exceeding_threshold + permanent_debt_allowed_minor;
+        let minimal_acceptable_payment = exceeding_threshold + permanent_debt_allowed_minor;
+
+        let condition_of_debt_fast_growth = minimal_acceptable_payment
+            >= Self::FIRST_QUALIFICATION_CONDITION_COEFFICIENT * considered_forgiven;
 
-        let first_condition =
-            minimal_payment_accepted >= Self::FIRST_CONDITION_COEFFICIENT * considered_forgiven;
+        let condition_of_position_on_rather_the_left_half_of_the_slope = considered_forgiven
```

Comment:
Let' call it differently: `situated_on_the_left_half_of_the_slope`. (your suggestion, quite nice)